src/core/obj.js

```diff
@@ -200,6 +201,32 @@ class Catalog {
     return markInfo;
   }
 
+  get structTreeRoot() {
+    let structTree = null;
+    try {
+      structTree = this._readStructTreeRoot();
+    } catch (ex) {
+      if (ex instanceof MissingDataException) {
+        throw ex;
+      }
+      warn("Unable to structTreeRoot info.");
```

Comment:
Nit: Missing word here?
```suggestion
      warn("Unable to read structTreeRoot info.");
```